test/e2e/service.go

```diff
@@ -2764,3 +2821,23 @@ func describeSvc(ns string) {
 		"describe", "svc", fmt.Sprintf("--namespace=%v", ns))
 	framework.Logf(desc)
 }
+
+func checkReachability(expectToBeReachable bool, namespace, pod, target string) {
```

Comment:
Is this function name too general? May be change it to something like checkReachabilityFromPod?